packages/client/screens/MusicPlaylistEditorMusicTrackVoteCreationForm/__tests__/ExportToMtv.test.tsx

```diff
@@ -559,8 +559,10 @@ describe('Export MPE room to MTV room', () => {
             plan.paths.forEach((path) => {
                 it(path.description, async () => {
                     const fakeMpeRoom = db.searchableMpeRooms.create();
+                    const { roomID, roomName } = fakeMpeRoom;
                     const fakeRoomState = generateMpeWorkflowState({
-                        ...fakeMpeRoom,
```

Comment:
This caused an issue because `roomName` and `name` are not the same properties?
Sorry for this dark bug.

Reply:
not sure, some dark magic happened in this file